### ISSUE TITLE
Convert all formal parameters to named doesn't handle super parameters well

### ISSUE THREAD

**Issue by bwilkerson** [dart-lang/sdk#55068]
I'm not sure whether there's anything we can reasonably do here, but given the code

```dart
class A {
  int a;
  A(this.a);
}

class B extends A {
  B(super.a);
}
```

If you select the constructor for `A` and ask to "Convert all formal parameters to named", the parameter "super.a" won't be changed and results in a compilation error.

Selecting the constructor for `B` and performing the same operation will fix the code, but it isn't clear that the constructor in `B` should be modified when refactoring the constructor in `A`.

Performing the refactorings in the other order (converting `B` first and then `A`) will also result in compilation errors until both have been converted.

**Comment by bwilkerson** [comment]
@scheglov In case you have ideas (other than "use a real IDE" 😄 )

**Comment by scheglov** [comment]
Yes, support for constructors, specifically for super formal parameters is not fully ready.
I will look at these when I have more spare time.

**Comment by scheglov** [comment]
https://dart-review.googlesource.com/c/sdk/+/355505 should make sure that we don't suggest partial changes.
